app/assignment/directives/instructorAssignmentTable.js

```diff
@@ -189,13 +192,27 @@ let instructorAssignmentTable = function ($rootScope, AssignmentActionCreators,
 							}
 
 							unassignedTermCodes[sectionGroup.termCode].push(sectionGroup.id);
+						} else {
+							sectionGroup.teachingAssignmentIds.forEach(function(teachingAssignmentId) {
+								var teachingAssignment = scope.view.state.teachingAssignments.list[teachingAssignmentId];
+								if (!teachingAssignment.instructorId && teachingAssignment.instructorTypeId) {
+									unassignedInstructorTypes[teachingAssignment.instructorTypeId] = unassignedInstructorTypes[teachingAssignment.instructorTypeId] || {};
+									unassignedInstructorTypes[teachingAssignment.instructorTypeId][teachingAssignment.termCode] = unassignedInstructorTypes[teachingAssignment.instructorTypeId][teachingAssignment.termCode] || [];
+									unassignedInstructorTypes[teachingAssignment.instructorTypeId][teachingAssignment.termCode].push(teachingAssignment.sectionGroupId);
+									unassignedInstructorTypeLength += 1;
+
+									if (unassignedInstructorTypeIds.indexOf(teachingAssignment.instructorTypeId) == -1) {
+										unassignedInstructorTypeIds.push(teachingAssignment.instructorTypeId);
+									}
+								}
+							});
 						}
 					});
 
 					var theStaffLength = _.keys(scope.view.state.theStaff.termCodes).length;
 					var unassignedLength = _.keys(unassignedTermCodes).length;
 
-					if ((theStaffLength == 0) && (unassignedLength == 0)) {
+					if ((theStaffLength == 0) && (unassignedLength == 0) && unassignedInstructorTypeLength == 0) {
```

Comment:
Always match the styling of the existing code, unless it is outside the code guidelines. In this case, the two other conditionals are enclosed in parentheses, and you introduced a new style where they're not.